operations/network/management_network/canu/custom_config.md

```diff
@@ -0,0 +1,146 @@
+sw-spine-002:  |
+    ip route 0.0.0.0/0 10.103.15.189
+    interface 1/1/36
+        no shutdown
+        ip address 10.103.15.190/30
+        exit
+    system interface-group 3 speed 10g
+    snmp-server vrf default
+    snmpv3 user testuser auth md5 auth-pass plaintext xxxxxx priv des priv-pass plaintext xxxxx
+sw-leaf-bmc-001:  |
+    interface 1/1/20
+        no routing
+        vlan access 4
+        spanning-tree bpdu-guard
+        spanning-tree port-type admin-edge
+    snmp-server vrf default
+    snmpv3 user testuser auth md5 auth-pass plaintext xxxxxx priv des priv-pass plaintext xxxxx
+```
+
+Mellanox/Dell
+
+```text
+sw-spine-001:  |
+    interface ethernet 1/1 speed 10G force
+    interface ethernet 1/1 description "sw-spine02-1/16"
+    interface ethernet 1/1 no switchport force
+    interface ethernet 1/1 ip address 10.102.255.14/30 primary
+    interface ethernet 1/1 dcb priority-flow-control mode on force
+    ip route vrf default 0.0.0.0/0 10.102.255.13
+    snmp-server user testuser v3 capability admin
+    snmp-server user testuser v3 enable
+    snmp-server user testuser v3 enable sets
+    snmp-server user testuser v3 encrypted auth md5 xxxxxxx priv des xxxxxxx
+    snmp-server user testuser v3 require-privacy
+sw-spine-002:  |
+    interface ethernet 1/16 speed 10G force
+    interface ethernet 1/16 description "sw-spine01-1/16"
+    interface ethernet 1/16 no switchport force
+    interface ethernet 1/16 ip address 10.102.255.34/30 primary
+    interface ethernet 1/16 dcb priority-flow-control mode on force
+    ip route vrf default 0.0.0.0/0 10.102.255.33
+    snmp-server user testuser v3 capability admin
+    snmp-server user testuser v3 enable
+    snmp-server user testuser v3 enable sets
+    snmp-server user testuser v3 encrypted auth md5 xxxxxxx priv des xxxxxxx
+    snmp-server user testuser v3 require-privacy
+sw-leaf-bmc-001:  |
+    interface ethernet1/1/12
+      description sw-leaf-bmc-001:12==>cn003:2
+      no shutdown
+      switchport access vlan 4
+      mtu 9216
+      flowcontrol receive off
+      flowcontrol transmit off
+      spanning-tree bpduguard enable
+      spanning-tree port type edge
+    interface vlan7
+        description CMN
+        no shutdown
+        ip vrf forwarding Customer
+        mtu 9216
+        ip address 10.102.4.100/25
+        ip access-group cmn-can in
+        ip access-group cmn-can out
+        ip ospf 2 area 0.0.0.0
+    snmp-server group cray-reds-group 3 noauth read cray-reds-view
+    snmp-server user testuser cray-reds-group 3 auth md5 xxxxxxxx priv des xxxxxxx
+    snmp-server view cray-reds-view 1.3.6.1.2 included
+```
+
+To generate switch configuration with custom config injection.
+
+```text
+canu generate switch config --csm 1.2 -a full --shcd FILENAME.xlsx --tabs INTER_SWITCH_LINKS,NON_COMPUTE_NODES,HARDWARE_MANAGEMENT,COMPUTE_NODES --corners J14,T44,J14,T48,J14,T24,J14,T23 --sls-file SLS_FILE --name sw-spine-001 --custom-config CUSTOM_CONFIG_FILE.yaml
```

Comment:
I'm mainly thinking of what to safeguard here so someone knows they have to adapt this entire command to their use case and not just the FILENAME.xlsx and CUSTOM_CONFIG_FILE.yaml parameters.

I presume these are just examples, but replacing the CSM version string of 1.2 with a placeholder will serve as a good reminder for folks to fill it in.

Secondly, is there someway for users to identify they have to figure out the corners and tabs?

I know in some of the other canu pages there are helpers for the these arguments, for example some are scattered about here: https://github.com/Cray-HPE/docs-csm/blob/main/operations/network/management_network/validate_shcd.md#validation-steps 

Maybe that page can be paraphrased, or there's a better reference, or a note can just be added here to read the man page.